index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <div>
+      <h1>Dr. Norman Borlaug</h1>
+      <p>The man who saved a billion lives</p>
+    </div>
```

Comment:
I don't think using a `div` tag for a header is a good idea at all, always use semantic HTML tags

Reply:
- [ ] Kindly, make sure your comments are kind & polite because that'll help students in experiencing less stress & frustration when implementing changes. I suggest using words like `please`, `kindly`, and `I suggest`.

- [ ] I recommend following the suggestions I made in my previous comment as well :blush:.

- [ ] [OPTIONAL] I suggest using emojis and gifs, they make your code review more human-friendly.